doc/source/user_guide/missing_data.rst

```diff
@@ -659,6 +659,7 @@ Pass nested dictionaries of regular expressions that use the ``regex`` keyword.
 Pass a list of regular expressions that will replace matches with a scalar.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
Can these examples be changed to not produce a warning?

Reply:
updated to not produce warning + green